static/app/utils/theme.tsx

```diff
@@ -998,7 +1232,7 @@ export const lightTheme = {
   superuserSidebar: '#880808',
 };
 
-export const darkTheme: Theme = {
+export const darkTheme = {
```

Comment:
Do we want to add `satisfies` here?

Reply:
Let me add back the type, I made a mistake here